src/Flicking.ts

```diff
@@ -1511,6 +1520,7 @@ class Flicking extends Component<FlickingEvents> {
     camera.updateAdaptiveHeight();
     camera.updateOffset();
     await renderer.render();
+    this._updatePanelOrder(camera);
```

Comment:
I think it would be better to move to the camera class method.

Reply:
I have modified `updatePanelOrder` to camera class method.
You can check the most recent commit.
`updatePanelOrder` requires `flicking.control.controller.updateDirection();` to be called.
Should this be the camera class method, even though it interferes with flicking's controller?